<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(85, 85, 85);font-size:13px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="ShowHideHFrame">
     <layout class="QHBoxLayout" name="showHideShortcuts">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="showHideLabel">
        <property name="text">
         <string>Show/Hide Toggl</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="showHIdeHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="showHideButton">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Record Shortcut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showHideClear">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear Shortcut</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ContinueStopHFrame">
     <layout class="QHBoxLayout" name="continueStopShortcuts">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="continueStopLabel">
        <property name="text">
         <string>Continue/Stop timer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="continueStopHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="continueStopButton">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Record Shortcut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="continueStopClear">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear shortcut</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useSystemProxySettings">
     <property name="styleSheet">
      <string notr="true">color:rgb(85, 85, 85);</string>
     </property>
     <property name="text">
      <string>Use system proxy settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="useProxy">
     <property name="title">
      <string>Use pro&amp;xy to connect to Toggl</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>Host</string>
        </property>
        <property name="buddy">
         <cstring>idleDetectionSettings</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="proxyHost"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="proxyPort"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="useranameLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="proxyUsername"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="proxyPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="idleDetectionSettings">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="idleDetection">
        <property name="styleSheet">
         <string notr="true">color:rgb(85, 85, 85);</string>
        </property>
        <property name="text">
         <string>Idle detection</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="idleMinutes">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="idleMinutesLabel">
        <property name="text">
         <string>minutes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="recordTimeline">
     <property name="styleSheet">
      <string notr="true">color:rgb(85, 85, 85);</string>
     </property>
     <property name="text">
      <string>Record timeline</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="TrackingReminderHFrame">
     <layout class="QHBoxLayout" name="reminderSettings">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="remindToTrackTime">
        <property name="styleSheet">
         <string notr="true">color:rgb(85, 85, 85);</string>
        </property>
        <property name="text">
         <string>Remind to track time</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="reminderHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="reminderMinutes">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="reminderMinutesLabel">
        <property name="text">
         <string>minutes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="PomodoroHFrame">
     <layout class="QHBoxLayout" name="pomodoroSettings">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="pomodoroTimer">
        <property name="styleSheet">
         <string notr="true">color:rgb(85, 85, 85);</string>
        </property>
        <property name="text">
         <string>Pomodoro timer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="pomodoroHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="pomodoroMinutes">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>25</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pomodoroMinutesLabel">
        <property name="text">
         <string>minutes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="PomodoroBreakHFrame">
     <layout class="QHBoxLayout" name="pomodoroBreakSettings">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="pomodoroBreakTimer">
        <property name="styleSheet">
         <string notr="true">color:rgb(85, 85, 85);</string>
        </property>
        <property name="text">
         <string>Pomodoro break timer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="pomodoroBreakHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="pomodoroBreakMinutes">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>25</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pomodoroBreakMinutesLabel">
        <property name="text">
         <string>minutes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="focusAppOnShortcut">
     <property name="styleSheet">
      <string notr="true">color:rgb(85, 85, 85);</string>
     </property>
     <property name="text">
      <string>Focus app on shortcut</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
